.github/workflows/fast.yml

```diff
@@ -0,0 +1,136 @@
+# Each job should typically run in under 5 minutes.
+name: fast
+
+on:
+  pull_request: {}
+
+env:
+  CARGO_INCREMENTAL: 0
+  CARGO_NET_RETRY: 10
+  RUST_BACKTRACE: short
+  RUSTUP_MAX_RETRIES: 10
+
+permissions:
+  contents: read
+
+jobs:
+
+  # Linting
+  check-clippy:
+    timeout-minutes: 10
```

Comment:
do you know if these configs can be set for all jobs in this file? could be nice to do now that we split them based on fast/slow...?

Reply:
afaict, no https://json.schemastore.org/github-workflow.json